### DISCUSSION TITLE
Notable contributions should have more filters

### DISCUSSION THREAD

**Discussion by Milo123459** [lowlighter/metrics#668] [Ideas]
Ideas for filters:
`commits` - the users commits
`commits_everyone` - the amount of commits in the repo, not just by the user
`pull_requests` - the amount of pull requests that user has opened
`issues` - the amount of issues that person has opened

**Comment by lowlighter** [comment]
Some were planned in #651 
I'll add these suggestion in it. Most of these will be gated through `plugin_followup_indepth` though

**Reply by lowlighter** [reply]
First two filters have shipped, the later may be added in the future when some work on integrating pr/issues in the notable contribution will be done 👍
